<html>
 <body>
  <hr/>
  <p>
   title: Home
   <br/>
   layout: default
  </p>
  <hr/>
  <h2>
   Deep Embedding for Single-cell Clustering
  </h2>
  <p>
   DESC is an unsupervised deep learning algorithm for clustering scRNA-seq data. The algorithm constructs a non-linear mapping function from the original scRNA-seq data space to a low-dimensional feature space by iteratively learning  cluster-specific gene expression representation and cluster assignments based on a deep neural network. This iterative procedure moves each cell to its nearest cluster, balances the biological and  technical differences between clusters, and reduces the influence of the batch effect. DESC also enables soft clustering by assigning cluster-specific probability to each cell, which facilitates the identification of cells clusters with high-confidence and interpretation of results.
   <br/>
   <img alt="DESC workflow" src="assets/images/desc_workflow.png"/>
  </p>
  <p>
   For thorough details, see the preprint:
   <a href="https://www.biorxiv.org">
    Bioxiv
   </a>
  </p>
  <p>
   <br/>
  </p>
  <h2>
   Usage
  </h2>
  <p>
   The
   <a href="https://github.com/eleozzr/desc">
    <strong>
     desc
    </strong>
   </a>
   package is an implementation of deep embedding for single-cell clustering. With desc, you can:
  </p>
  <ul>
   <li>
    Pre-process single cell gene expression data from various formats.
   </li>
   <li>
    Build a low-dimensional representation of the single-cell expression data.
   </li>
   <li>
    Obtain soft-clustering assignments of cells.
   </li>
   <li>
    Visualize the clustering results and gene expression patterns.
   </li>
  </ul>
  <p>
   <br/>
  </p>
  <h2>
   References
  </h2>
  <p>
   Please consider citing the references in case they are used.
   <br/>
   [1] DESC:
   <a href="https://www.biorxiv.org">
    Bioxiv
   </a>
   <br/>
   <br/>
  </p>
  <h2>
   Licence
  </h2>
  <p>
   Copyright (C) 2019 Mingyao's Lab
  </p>
 </body>
</html>